charts/tekton-operator/README.md

```diff
@@ -1,23 +1,29 @@
 # Tekton Operator Helm Chart
 
 This Helm chart installs the [Tekton Operator](https://tekton.dev/docs/operator/) into your Kubernetes (v1.16+) or Openshift cluster (v4.3+).
+Once this chart is published it can be installed directly, until then [helm-git](https://github.com/aslafy-z/helm-git) can be used.
 
 ## TL;DR
 
+Install [helm-git](https://github.com/aslafy-z/helm-git) plugin.
+
 ```sh
-git clone https://github.com/tektoncd/operator.git
+# Add the repo to helm (typically use a tag rather than master):
+helm repo add tektoncd "git+https://github.com/tektoncd/operator@charts?ref=master"
```

Comment:
Should probably be `main` instead of `master` for the ref.

Reply:
Fixed - thank you